schema/erc20/tokens.sql

```diff
@@ -143,12 +181,46 @@ COPY erc20.tokens (contract_address, symbol, decimals) FROM stdin;
 \\x94a2aaa374a8f2d52dad24330c8a0ec2934700ae	FCOUSDB	18
 \\x009e864923b49263c7f10d19b7f8ab7a9a5aad33	FKX	18
 \\xf04a8ac553fcedb5ba99a64799155826c136b0be	FLIXX	18
+\\x01fb4a9a68dd46852addfa98321def44289b7d8d	FNCD	18
+\\x04b7c9b88c1d33f0293449aa81b143272bcb2a8b	FNCD	18
+\\x07be1d613ec9a7965bbe98064e66e3b3d648da66	FNCD	18
+\\x151a1eb5a1c989cdca7a48c07a1a5ae0ef80ea0d	FNCD	18
+\\x15887c0a2ee20e2b6f22a977b18bce47c5c5a2f2	FNCD	18
+\\x2561df6c7ee681864c8a1245aa51a06a1e22a52b	FNCD	18
+\\x26157a6e4f84c49942e32484d4728aa6e05c6417	FNCD	18
+\\x304b137223b0b240ba9e86d49aafc152a11b6dc2	FNCD	18
+\\x3796878de516db33cf1cba0b7d333cd1db96434e	FNCD	18
+\\x3798e0b7453b2954d61cbc86bbea2d52292531bf	FNCD	18
+\\x536cb3bafccb17537ba581b0ac4355f7e19df395	FNCD	18
+\\x6ec5a9d14d5206cad15b32e4bbe5e0b0be30ce14	FNCD	18
+\\x84a9986f4d04e73bc9442245b8dfef7f77305637	FNCD	18
+\\x8e0f05cd0e4e9dc28dbba761aac1f59337d1c455	FNCD	18
+\\x910822f6844304afb34ae8d7db619cdfa498ad49	FNCD	18
+\\x994e4008bcdc6cf08094fd08bb4418e70b6d686d	FNCD	18
+\\x9fd85ca57f20e7a39726e0e9cfad51f45bb2b0e3	FNCD	18
+\\xa0bbe61f8705d0dde7fa46f78a148c913b260c35	FNCD	18
+\\xb2db902091500cd04f41396d664dfa653517af68	FNCD	18
+\\xb7f5497f2ae792d59608c9a046b141cd29f3b526	FNCD	18
+\\xbfc615ab171d10a4e6498549cd3a46766feefe01	FNCD	18
+\\xc3308318eca1eab76d2f4b52c310289ae526c8ca	FNCD	18
+\\xce6437ee4932cfba3ecaefb93a1929a379b13b24	FNCD	18
+\\xcf97b4f961fa247cd60eb3cbf92a135fa2c9cc5d	FNCD	18
+\\xd2a3e62ca1658421a8a711bd1f8ed9d01780e6e6	FNCD	18
+\\xd609609fdf3a6ea89d5ea981af5ce5c3c3daa44a	FNCD	18
+\\xe23a3a78b131553c0bab4323bba7f61f6950bdaa	FNCD	18
+\\xf0fd2500bd31a56cf1a3839e32fd28e11cda60b3	FNCD	18
+\\xfb6f6467724b7cdc75a1977812d78b7467f69d77	FNCD	18
+\\xfc382c9f9f65a66bbd3b35e10e7ac5a7abf11c8f	FNCD	18
```

Comment:
Are these different versions of the same token or what's going on here?

Reply:
Those are indeed different versions of the same token. Don't know how useful there are but there are in the list of tokens traded (at least once) on bancor.